### PR TITLE
Update entry-context.md

### PR DESCRIPTION
**Pull request by dear-lizhihua**
1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/writers-guide/).
3. Read through the PR diff carefully as sometimes this can reveal issues. The work will be reviewed, but this can save some effort.
4. Remove these instructions from your PR as they are for your eyes only.

### CODE REVIEW

File: content/configuration/entry-context.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The base directory, an **absolute path**, for resolving entry points and loaders
 context: path.resolve(__dirname, "app")
 ```
 
-By default the current directory is used, but it's recommended to pass a value in your configuration. This makes your configuration independent from CWD.
+By default the current directory is used, but it's recommended to pass a value in your configuration. This makes your configuration independent from CWD(current working directory).
```
Comment (simon04) [inline content/configuration/entry-context.md]: Add a space before the opening parenthesis.

**Comment by dear-lizhihua** [comment]
oh, sorry.
like this:
CWD (current working directory)

**Comment by simon04** [comment]
Thank you. Now only the CLA is outstanding, see https://github.com/webpack/webpack.js.org/pull/830#issuecomment-277595516

**Comment by dear-lizhihua** [comment]
ok, do you mean "CLA signed"?

**Comment by simon04** [comment]
Yes. Thank you!

**Comment by dear-lizhihua** [comment]
thank you, too!

**Pull request merged**